zebra-chain/src/orchard/commitment.rs

```diff
@@ -151,11 +151,25 @@ impl NoteCommitment {
         let psi_bytes: [u8; 32] = psi.into();
 
         // g*d || pk*d || I2LEBSP_64(v) || I2LEBSP_l^Orchard_Base(ρ) || I2LEBSP_l^Orchard_base(ψ)
-        s.append(&mut BitVec::<Lsb0, u8>::from_slice(&g_d_bytes[..]));
-        s.append(&mut BitVec::<Lsb0, u8>::from_slice(&pk_d_bytes[..]));
-        s.append(&mut BitVec::<Lsb0, u8>::from_slice(&v_bytes[..]));
-        s.append(&mut BitVec::<Lsb0, u8>::from_slice(&rho_bytes[..]));
-        s.append(&mut BitVec::<Lsb0, u8>::from_slice(&psi_bytes[..]));
+        s.append(
+            &mut BitVec::<Lsb0, u8>::from_slice(&g_d_bytes[..])
+                .expect("must work for small slices"),
+        );
+        s.append(
+            &mut BitVec::<Lsb0, u8>::from_slice(&pk_d_bytes[..])
+                .expect("must work for small slices"),
+        );
+        s.append(
+            &mut BitVec::<Lsb0, u8>::from_slice(&v_bytes[..]).expect("must work for small slices"),
+        );
+        s.append(
+            &mut BitVec::<Lsb0, u8>::from_slice(&rho_bytes[..])
+                .expect("must work for small slices"),
+        );
+        s.append(
+            &mut BitVec::<Lsb0, u8>::from_slice(&psi_bytes[..])
+                .expect("must work for small slices"),
+        );
```

Comment:
I think that with the new Rust version's support for array `IntoIterator`, this could be simplified to:

```suggestion
        s.extend(g_d_bytes);
        s.extend(pk_d_bytes);
        s.extend(v_bytes);
        s.extend(rho_bytes);
        s.extend(psi_bytes);
```